internal/routes/server.go

```diff
@@ -359,6 +322,78 @@ func Server(
 	return mux, nil
 }
 
+// codestatusRoutes are the routes for checking code statuses.
```

Comment:
The camel casing on a few of these is unusual?

Reply:
It's matching the package name